### PR TITLE
Add configuration option to change log file location

### PR DESCRIPTION
**Pull request by gsloan-qs**
This change is [Reviewable](https://reviewable.io/reviews/qualisystems/devguide_source/62)

### CODE REVIEW

File: _shells/tips-and-tricks.md [hunk 1]
```diff
@@ -53,6 +53,14 @@ relative to the driver in the virtual environment location at:
 
 Under windows, [venv] will be located at _%programdata%\\qualisystems\\venv_.
 
+The Execution Server can be configured so that all drivers log to the same locaiton rather than under their own virtual environmaent location. Add the following key to the ES _customer.config_:
+
+{% highlight xml %}
+<add key="DefaultPythonEnvrionmentVariables" value="LOG_PATH=c:\[path]\[to]\[directory]"/>
```
Comment (arueth) [inline _shells/tips-and-tricks.md]: Is this suppose to be DefaultPythonEnv**ri**onmentVariables or DefaultPythonEnv**ir**onmentVariables

Comment (gsloan-qs) [inline _shells/tips-and-tricks.md]: Depends on what you mean by "supposed to be". As we found out today, it only works if misspelled.  Looks like a bug against the execution server needs to be opened.

**Comment by TomerAdmon** [comment]
@Shnitselon - Please take a look, if needed we will open a bug.

**Comment by TomerAdmon** [comment]
Hi, 
This key is a hidden key and not recommended for customers use.
Since 8.0 all python driver logs should use cloudshell-core and then all logs will be collected under a folder with the name of the sandbox-Id which is the command.

closing....

**Pull request closed without merging**